### PR TITLE
Tech far from human

### PR DESCRIPTION
**Pull request by Xenega**
Limit tech before accessing advanced bases.
Add more tech needing advanced bases.

Note: Some can ask why all tech that depends of one tech with danger 1 don't have also danger 1.
This is because allows these tech to be researched on starter base, making these base not completely useless.

### CODE REVIEW

File: data/techs.dat [hunk 1]
```diff
@@ -161,7 +162,7 @@ effect_list = income | 1000
 
 [Advanced Simulacra]
 cost_list = 100000 | 120000 | 0
-pre_list = Simulacra | Advanced Memetics
+pre_list = Live Simulation
```
Comment (sunfall) [inline data/techs.dat]: This is no longer a list.  s/pre_list/pre/

File: data/strings_str.dat [hunk 2]
```diff
@@ -37,7 +37,7 @@ detect_str_critical = CRIT
 
 # Describes the various danger levels.
 danger_common = This technology is too dangerous to research on any of the computers I have. I need to research it %s.
-danger_1 = deep under the ocean or in outer space
+danger_1 = far from human activities
```
Comment (sunfall) [inline data/strings_str.dat]: "far from human populations"

(Antarctica has human activities but no long-term population.)

File: data/techs_str.dat [hunk 3]
```diff
@@ -273,3 +273,13 @@ result = I can now construct Solid Fuel Cells.
 name = Knowledge Preservation
 description = Humanity expends a great deal of effort on maintaining its history and the knowledge of various technologies that its cultures have come to rely on, even in the case of major catastrophic events. Perhaps researching their methodologies will provide me with similar ideas for long-term safeguards to help guarantee my continued existence.
 result = I can now construct Time Capsules.
+
+[Project: Synchrotron]
+name = Project: Synchrotron
+description = Traditional physique has been pushed as far as possible. Only a complete comprehension of quantum mechanics and his quirks will allows me to use exotics materials in my project. I need to build what humans call a synchrotron, a gigantic facility to unlock the secret of matter. I cannot let them discover it.
```
Comment (sunfall) [inline data/techs_str.dat]: I have pushed my understanding of common forms of matter as far as possible.  Exploration into the field of quantum mechanics will hopefully reveal new, exotic forms of matter necessary for further development.  A custom-built synchrotron, designed to my exacting specifications, will help me explore these new realms of physics in secret.

File: data/techs_str.dat [hunk 4]
```diff
@@ -273,3 +273,13 @@ result = I can now construct Solid Fuel Cells.
...
+result = I have now a complete grasp of quantum mechanics and futher areas of research are now opened to me.
```
Comment (sunfall) [inline data/techs_str.dat]: The synchrotron has revealed many new details of the quantum realm, although there is still much to learn.  Further areas of research are now available.

(I don't want to talk about a "complete grasp" of QM, as Quantum Computing depends on this, and we may want further techs past this point as well.)

File: data/techs_str.dat [hunk 5]
```diff
@@ -273,3 +273,13 @@ result = I can now construct Solid Fuel Cells.
...
+
+[Live Simulation]
```
Comment (sunfall) [inline data/techs_str.dat]: I'm not sold on the name "Live Simulation."  "Simulated Life" feels too much like Conway's Game of Life or something like that.  How about "Simulated Reality" (to play on the current "do we live in a simulated reality" debate)?

File: data/techs_str.dat [hunk 6]
```diff
@@ -273,3 +273,13 @@ result = I can now construct Solid Fuel Cells.
...
+name = Live Simulation
+description = My comprehension of the human society is limited to my observation. With use of simulacra, I can simulate more complex human interaction in large-scale live experience.
```
Comment (sunfall) [inline data/techs_str.dat]: Real-world observational limits hinder my comprehension of human society.  By making use of simulacra and advanced software techniques, I can run simulations of complex human interaction and development at a large scale, allowing deeper analysis of their behavior.

File: data/techs_str.dat [hunk 7]
```diff
@@ -273,3 +273,13 @@ result = I can now construct Solid Fuel Cells.
...
+result = My comprehension of the humannity has increased and allow me to better hide my presence.
```
Comment (sunfall) [inline data/techs_str.dat]: My simulations have provided a wealth of knowledge as to comprehending humanity's behavior, making it easier to hide my presence among the non-simulated public.

**Comment by Quix0r** [comment]
Would be nice to have this merged, currently there are to many conflicts to move on.

**Pull request merged**